src/app_model/types/_keys/_keybindings.py

```diff
@@ -136,10 +138,60 @@ class KeyBinding(BaseModel):
     the two keypress codes with a space. For example, 'Ctrl+K Ctrl+C'.
     """
 
-    parts: List[SimpleKeyBinding] = Field(..., min_items=1)
+    __root__: str
+    _parts: List[SimpleKeyBinding] = PrivateAttr()
+
+    @classmethod
+    def _parts_error(cls, exc: Exception) -> ValidationError:
+        return ValidationError(
+            errors=[ErrorWrapper(exc=exc, loc=("parts",))], model=cls
+        )
+
+    def __init__(self, **data: Any) -> None:
```

Comment:
Optional: any strong reason to use `**data` instead of expanding the two known parameters `__root__` and `parts` as named keyword arguments with typing info?